### ISSUE TITLE
[Unstable 0.10.2.0] Submarine price doesn't seem to be affected when removing or adding items to it after the initial price has been set

### ISSUE THREAD

**Issue by Rokvach** [Regalis11/Barotrauma#3296]
Repro steps:
1. Make a copy of a vanilla sub or a custom sub
2. Check the price of the submarine
3. Remove the entire submarine

OR

3. Spam a lot of new items to the submarine
4. Go save the submarine and see that the price hasn't changed at all

I tested by restarting the game and other means to try and force it to update but it seems that the sub will stay priced depending of the price it has been given when it was first saved.

**Comment by Regalis11** [comment]
Note that we only set a _minimum_ price for the submarine based on the volume of the hulls and the value of the items: they don't affect the price of the sub directly as long as the price is above the minimum.

Did a bit of testing and couldn't find any issues with this.
1. Removed all hulls and items
2. The minimum price of the sub is 1000 (the lowest possible price), as expected .

OR

1. Loaded Dugong (minimum price 3951)
2. Added a ton of nuclear shells in the sub
3. Minimum price is now over 5000

**Comment by Rokvach** [comment]
Aa now I got it, I thought that the price set for the Dugong and couple custom subs I tested was already the minimum price, but it wasn't. So going past the manually defined price does update the price of the sub rather than add to the manually defined price. 

I also was testing this by trying to add new devices or walls to the submarine, but its only the actual items which affects it like the nuclear shells. 

Yeah no issues here then, closing.
